iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/TransportRoutePreviewStatus.swift

```diff
@@ -3,6 +3,7 @@ final class TransportRoutePreviewStatus: SolidTouchView {
   @IBOutlet private weak var etaLabel: UILabel!
   @IBOutlet private weak var stepsCollectionView: TransportTransitStepsCollectionView!
   @IBOutlet private weak var stepsCollectionViewHeight: NSLayoutConstraint!
+  @IBOutlet private weak var stepsCollectionScrollView: UIScrollView?
```

Comment:
Why is it ? instead of ! ?

Reply:
`Steps Collection View` is included in `MWMiPhoneRoutePreview.xib` and `MWMiPadRoutePreview.xib`. I added scroller only to iPhone layout. So for iPad it should be nil.

**But** I tested on iPad simulator - and lower panel is scrollable. How to test `MWMiPadRoutePreview.xib`?